src/frontends/common/src/utils.cpp

```diff
@@ -42,7 +42,7 @@ static std::string _get_frontend_library_path() {
 #elif defined(__APPLE__) || defined(__linux__)
     Dl_info info;
     dladdr(reinterpret_cast<void*>(ov::frontend::get_frontend_library_path), &info);
-    return ov::util::get_directory(std::string(info.dli_fname)).c_str();
+    return ov::util::get_directory(ov::util::get_absolute_file_path(std::string(info.dli_fname))).c_str();
```

Comment:
should not we just reuse https://github.com/openvinotoolkit/openvino/blob/master/src/common/util/src/file_util.cpp#L315-L335 and remove this one? Looks like function from ov::util already has `get_absolute_file_path`

Reply:
Ping

Reply:
hi @ilya-lavrenov, do you mean to create a common function like get_library_path() that get_ov_library_path and get_frontend_library_path can call it. I also see a pr [12720](https://github.com/openvinotoolkit/openvino/pull/12720) which is also relate to get_fontend_library_path. so if want to reuse the code, maybe need to refactor the code

Reply:
I wanted to say that `_get_frontend_library_path` is the same as `get_ov_library_path_a`. Can we remove first one and use the second one?

Reply:
1. the function get_ov_library_path_a is static, so if want to reuse it, need to remove static.
2. if frontend library and ov library are the same library, it will be ok to reused the function.
from [12720](https://github.com/openvinotoolkit/openvino/pull/12720)， get_frontend_library_path also need to do wth "std::wstring", so Maybe We could merger prs to fix issues and then create another pr to do code optimization

Reply:
we could reuse ov::util::get_ov_lib_path() which is public
ok, let's merge first.